crates/modelardb_server/tests/integration_test.rs

```diff
@@ -54,7 +54,7 @@ const TABLE_NAME: &str = "table_name";
 const HOST: &str = "127.0.0.1";
 
 /// The next port to be used for the server in an integration test. Each test uses a unique port and
-/// local data folder so they can run in parallel and so that any failing tests does not cascade.
+/// local data folder, so they can run in parallel and so that any failing tests does not cascade.
```

Comment:
"any failing tests does not" -> "failing tests do not"

Reply:
Fixed.